src/Core/ViewHelper/AbstractTagBasedViewHelper.php

```diff
@@ -76,18 +76,12 @@ public function initializeArguments()
     }
 
     /**
-     * Sets the tag name to $this->tagName.
-     * Additionally, sets all tag attributes which were registered in
-     * $this->tagAttributes and additionalArguments.
-     *
-     * Will be invoked just before the render method.
-     *
-     * @return void
-     * @api
+     * @param array $arguments
      */
-    public function initialize()
+    public function setArguments(array $arguments)
```

Comment:
Can you move this to a separate PR please?

Reply:
Will do - by the way, this one relates to a coming change which cleans up the lifecycle methods we no longer use / we can get rid of easily.